src/logic/test/operate.test.js

```diff
@@ -0,0 +1,17 @@
+import operate from '../operate';
+
```

Comment:
- [ ] You are missing to test the division by 0 output as well as the % mod operation, please do so to comply with the project requirements.